include/wx/gtk/webview_webkit.h

```diff
@@ -151,6 +155,11 @@ class WXDLLIMPEXP_WEBVIEW wxWebViewWebKit : public wxWebView
     //create-web-view signal and so we need to send a new window event
     bool m_creating;
 
+#if wxUSE_WEBVIEW_WEBKIT2
+    // This methods needs to be public to make it callable from a callback
```

Comment:
```suggestion
    // This method needs to be public to make it callable from a callback
```

I wonder if we might also want to rename it to start with `WX` prefix to avoid any conflicts with the application methods.